### ISSUE TITLE
Color Contrast: Discussion Label

### ISSUE THREAD

**Issue by AutoSponge** [Hashnode/user-feedback#171]
**Describe the bug**
Element has insufficient color contrast of 3.12 (foreground color: `#ffffff`, background color: `#2196f3`, font size: 9.8pt (13px), font weight: bold).

**To Reproduce**

**Expected behavior**
Expected contrast ratio of 4.5:1

Element location:
```css
.hn-card:nth-child(5) > .post-author.justify-content-between.flex-row > .author-info.flex-grow-1.flex-row > .flex-wrap.flex-grow-1.flex-row > .discuss-label
```

Element source: `<span class="discuss-label">Discussion</span>`

**Additional context**
Possible fixes:
- switch background to palette color `#2962FF` for 4.9:1
- switch to darker version of `#2196f3`
- switch text-color to black for 6.7:1 ratio

**Comment by BolajiAyodeji** [comment]
Hi @AutoSponge,

Thanks for the amazing feedback. We are working on getting this fixed and we'll update you soon.
